ENH: Add `cmasher.combine_cmaps` (#122)
1313e/CMasher: docs/source/user/usage.rst

    Application overview plot of *CMasher*'s colormaps.

.. _combine_colormaps:

Combine colormaps
-----------------

> *CMasher* offers a utility function :func:`~cmasher.combine_cmaps`, which enables the combination of multiple colormaps at specified ``nodes`` (where a node denotes the point separating adjacent colormaps). You can directly pass several colormaps using the function like so :pycode:`combine_cmaps("cmr.rainforest", "cmr.torch_r")`. By default, each sub-colormap occupies an equal portion of the final colormap.

Reviewer: ```suggestion
*CMasher* offers a utility function :func:`~cmasher.combine_cmaps`, which enables the combination of multiple colormaps at specified ``nodes`` (where a node denotes the point separating adjacent colormaps, within the interval ]0, 1[). You can directly pass several colormaps using the function like so :pycode:`combine_cmaps("cmr.rainforest", "cmr.torch_r")`. By default, each sub-colormap occupies an equal portion of the final colormap.
```

.. figure:: images/combine_cmaps_equal.png
    :alt: Combine two colormaps with default equal separation.
    :width: 100%
    :align: center

    Combine two colormaps with default equal separation.

> Alternatively, you could also manually specify the ``nodes``. For example :pycode:`cmr.combine_cmaps("cmr.rainforest", "cmr.torch_r", nodes=[0.75])` would allocate the starting 75% of the final colormap to "cmr.rainforest"and the remaining 25% to "cmr.torch_r".

Reviewer: ```suggestion
Alternatively, you may want to specify ``nodes`` explicitly. For example :pycode:`cmr.combine_cmaps("cmr.rainforest", "cmr.torch_r", nodes=[0.75])` would allocate the starting 75% of the final colormap to "cmr.rainforest"and the remaining 25% to "cmr.torch_r".
```
